src/Microsoft.EntityFrameworkCore.Tools.DotNet/Microsoft.EntityFrameworkCore.Tools.DotNet.csproj

```diff
@@ -7,9 +7,9 @@
     <OutputType>Exe</OutputType>
     <IsPackable>false</IsPackable>
     <RootNamespace>Microsoft.EntityFrameworkCore.Tools</RootNamespace>
+    <NetCoreAppImplicitPackageVersion>1.0.0</NetCoreAppImplicitPackageVersion>
```

Comment:
Here's what it does under the hood: https://github.com/dotnet/sdk/blob/42de009e12a80d2ee09e2ec13d6d30e430e1368d/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.props#L36-L38